io.openems.edge.batteryinverter.kaco.blueplanetgridsave/src/io/openems/edge/batteryinverter/kaco/blueplanetgridsave/statemachine/Undefined.java

```diff
@@ -0,0 +1,33 @@
+package io.openems.edge.batteryinverter.kaco.blueplanetgridsave.statemachine;
+
+import io.openems.edge.common.statemachine.StateHandler;
+
+public class Undefined extends StateHandler<State, Context> {
+
+	@Override
+	public State getNextState(Context context) {
+		switch (context.component.getStartStopTarget()) {
+		case UNDEFINED:
+			// Stuck in UNDEFINED State
```

Comment:
It can be undefined, when the config is set to "AUTO" and no `setStartStop()`command was given yet - in that case `private AtomicReference<StartStop> startStopTarget` is UNDEFINED.